src/conf.py

```diff
@@ -1330,13 +1331,17 @@ def defaultHttpHeaders(network, channel):
     try:
         language = supybot.protocols.http.requestLanguage.getSpecific(
                 network, channel)()
+        agent = random.choice(supybot.protocols.http.userAgent.getSpecific(
+                network, channel)())
     except registry.NonExistentRegistryEntry:
-        pass # Starting up; language will be set by HttpRequestLanguage later
+        pass
```

Comment:
Leave the comment, it's unclear why `NonExistentRegistryEntry` may be raised otherwise